### PR TITLE
Add Amazon DocumentDB Icon

### LINKED ISSUE #7726: AWS documentdb
### Brand Name

Amazon Web Services

### Website

https://aws.amazon.com/documentdb/

### Popularity Metric

AWS is #1 for tech

### Official Resources for Icon and Color

https://d1.awsstatic.com/webteam/architecture-icons/q3-2022/Asset-Package_07312022.e9f969935ef6aa73b775f3a4cd8c67af2a4cf51e.zip 

>> Architecture-Service-Icons >> Arch_Database >> 64 >> Arch_Amazon_DocumentDB_64.svg

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by RiteshK-611**
**Issue:** closes #7726

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
This commit will resolve issue #7726 by adding Amazon DocumentDB Logo in amazondocumentdb.svg and by updating simple-icons.json file

### Preview
amazondocumentdb

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'm seeing [quite a difference](https://github.com/simple-icons/simple-icons/assets/1830867/69f98d8e-0c1c-4e96-ae5b-a6442eb401c5) between your PR in black, and the source icon in red. Could you take another look please @RiteshK-611?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -551,6 +551,17 @@
                 ]
             }
         },
+        {
+            "title": "Amazon DocumentDB",
+            "hex": "C925D1",
+            "source": "https://aws.amazon.com/architecture/icons/",
+            "guidelines": "https://aws.amazon.com/architecture/icons/",
```
Comment (adamrusted) [inline _data/simple-icons.json:558]:
```suggestion
            "source": "https://aws.amazon.com/architecture/icons",
            "guidelines": "https://aws.amazon.com/architecture/icons",
```

File: icons/amazondocumentdb.svg
Comment (adamrusted) [inline icons/amazondocumentdb.svg:1]: I'm seeing some [minor differences](https://github.com/simple-icons/simple-icons/assets/1830867/dbfacbdc-217b-4307-8512-9975dc6572ad) in the more complex shapes. Could you re-export with a precision of 4 decimals please? 🙏

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/amazondocumentdb.svg
Comment (RiteshK-611) [reply icons/amazondocumentdb.svg:1]: Done!

Comment (adamrusted) [inline icons/amazondocumentdb.svg:1]: Icon still appears to be in 3 decimals of precision, and I'm still seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/bd7906f5-4929-4b77-a12a-9832f38d3824). Could you please regenerate from source (I'm using the 64*64 icons when comparing) with 4 decimals of precision? 🙏

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Review by adamrusted — CHANGES_REQUESTED** [review]
Seems you've overridden DynamoDB...

**Comment by RiteshK-611** [comment]
> Seems you've overridden DynamoDB...

oh.. yes... I fixed it
Thanks @adamrusted for being supportive

**Review by adamrusted — APPROVED** [review]
Happy with this, thanks @RiteshK-611 🎉

**Pull request merged**